contracts/interfaces/IERC2981Internal.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.8;
+
+/**
+ * @title ERC2981 interface
+ */
+interface IERC2981Internal {
+    error ERC2981Internal__RoyaltyExceedsMax();
```

Comment:
Errors should not be defined in the EIP interfaces in `interfaces/` directory.  Rather, they should be defined in an internal implementation interface, such as `IERC20BaseInternal`.

However, these contracts don't have a "base" implementation, so the pattern breaks down.  Closest analogue might be `ERC165`, which also includes a custom error but does not have a "base" implementation.  I'll think about what the best solution to this issue might be.

No action needed for now.